google-cloud-datastore/src/main/java/com/google/cloud/datastore/testing/LocalDatastoreHelper.java

```diff
@@ -193,14 +193,25 @@ public void start() throws IOException, InterruptedException {
   }
 
   /**
-   * Reset the internal state of the Datastore emulator.
+   * Resets the internal state of the Datastore emulator.
+   *
+   * <p>When running tests, one might {@code reset()} before each test, so earlier tests would not
+   * affect later ones.
    */
   public void reset() throws IOException {
     sendPostRequest("/reset");
   }
 
   /**
    * Stops the Datastore emulator.
+   *
+   * <p>It is important to stop the emulator. Since the emulator runs in its own process, not
+   * stopping it might cause it to become orphan.
+   *
+   * <p>It is not required to call {@link #reset()} before {@code stop()}.
+   *
+   * @param timeout The duration to wait for the emulator process to stop. It is recommended to set
+   *     this value high to ensure proper shutdown, like 5 seconds or more.
    */
   public void stop(Duration timeout) throws IOException, InterruptedException, TimeoutException {
```

Comment:
I'm not sure that this should be configureable. I don't think end users will have a better idea of how long it takes the emulator to shutdown than we do.

Reply:
Unfortunately it must be, it is required by the base class. I'll add a no-config overload.